build.gradle

```diff
@@ -252,7 +251,7 @@ publishing {
                 def dependenciesNode = asNode().appendNode('dependencies')
 
                 configurations.implementation.allDependencies.each {
-                    if (it.group != null && !it.group.contains("net.fabricmc") && it.name != null && it.name != "forge" && it.version != null) {
+                    if (it.group != null && !it.group.contains("net_fabricmc") && !it.group.contains("net.fabricmc") && it.name != null && it.name != "forge" && it.version != null) {
```

Comment:
When trying to use this dependency, it would often fail to resolve as it can't find the ``named`` version of it in your caches, this resolves that issue and lets Loom remap it